libc/test/CMakeLists.txt

```diff
@@ -8,10 +8,15 @@ add_custom_target(libc-long-running-tests)
 
 add_subdirectory(UnitTest)
 
-if(LIBC_TARGET_OS_IS_GPU AND
```

Comment:
Can we precommit or move this to a separate patch

Reply:
This is helpful for me working out why libc isn't running tests but otherwise unrelated to this PR, I'll drop it from the change and you can apply or discard it separately as you prefer.

Reply:
Done